common/sagas/schedule/currentSchedulingToggle.ts

```diff
@@ -0,0 +1,28 @@
+import { setGasLimitField } from 'actions/transaction/actionCreators/fields';
+import { call, put, takeLatest } from 'redux-saga/effects';
+import { SagaIterator } from 'redux-saga';
+import { TypeKeys } from 'actions/schedule/constants';
+import { SetGasLimitFieldAction } from 'actions/transaction';
+import { EAC_SCHEDULING_CONFIG } from 'libs/scheduling';
+import BN from 'bn.js';
+import { SetSchedulingToggleAction } from 'actions/schedule/actionTypes';
+
+export function* setGasLimitForScheduling({
+  payload: { value: useScheduling }
+}: SetSchedulingToggleAction): SagaIterator {
+  const gasLimit = useScheduling ? EAC_SCHEDULING_CONFIG.SCHEDULING_GAS_LIMIT : new BN('21000');
```

Comment:
Does it make sense to use EAC_SCHEDULING_CONFIG.SCHEDULE_GAS_LIMIT_FALLBACK for the else clause here instead? Would prefer having all of the constants in one place.